src/addrman.cpp

```diff
@@ -719,12 +719,21 @@ std::pair<CAddress, NodeSeconds> AddrManImpl::Select_(bool newOnly) const
     AssertLockHeld(cs);
 
     if (vRandom.empty()) return {};
-
     if (newOnly && nNew == 0) return {};
 
+    // Decide if we are going to search the new or tried table
+    bool search_tried;
```

Comment:
nit (for 9bf078f66c8f286e1ab5e34b8eeed7d80290a897):
```cpp
diff --git a/src/addrman.cpp b/src/addrman.cpp
index ec5b0213b..f608d60f0 100644
--- a/src/addrman.cpp
+++ b/src/addrman.cpp
@@ -722,15 +722,13 @@ std::pair<CAddress, NodeSeconds> AddrManImpl::Select_(bool newOnly) const
     if (newOnly && nNew == 0) return {};
 
     // Decide if we are going to search the new or tried table
-    bool search_tried;
+    bool search_tried{false};
 
     // Use a 50% chance for choosing between tried and new table entries.
     if (!newOnly &&
        (nTried > 0 &&
         (nNew == 0 || insecure_rand.randbool() == 0))) {
         search_tried = true;
-    } else {
-        search_tried = false;
     }
 
     if (search_tried) {
```

Reply:
Yes, can leave this way. This suggestion I made while reviewing the first commit but considering the whole context, you can leave it.